### ISSUE TITLE
Add the option to use Reddit's native image uploader when uploading images

### ISSUE THREAD

**Issue by dylanrees** [QuantumBadger/RedReader#402]
That way there's no need for an Imgur account.

**Comment by tjbortz1s** [comment]
I'm pretty sure the API for uploading to reddit is not public/documented/accessible yet.

**Comment by QuantumBadger** [comment]
Yep, there's no public API yet unfortunately.

But the app already lets you upload to imgur without an account :)

**Comment by dylanrees** [comment]
Yeah, using Imgur hasn't been a big deal.  Hopefully they'll release the API, though!
